utils/build/docker/nodejs/express4/iast/index.js

```diff
@@ -292,6 +292,16 @@ function initRoutes (app, tracer) {
     searchLdap(filter, req, res)
   })
 
+  app.get('/iast/hardcoded_secrets/test_insecure', (req, res) => {
```

Comment:
Can you add on docs/weblog a section explaining what this endpoint should be doing ?